### PR TITLE
Post-release 1694-dashboards

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

- Adds `creation_mode` to the dashboard object to track down if it was created from a template or duplicated from another dashboard (when we support this).
- Updates dashboard tests to DRF testing engine (and adds some minor improvements).
- Releases the new default app dashboard for everyone.
- Only shows the relevant empty message when on a dashboard (see related issue).
    
- Closes https://github.com/PostHog/retention/issues/18

## Checklist

- [X] All querysets/queries filter by Organization, by Team, and by User
- [X] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

**Comment by paolodamico** [comment]
CC @kpthatsme, relevant to activation. Also in case you want to review this PR yourself.

File: frontend/src/scenes/insights/EmptyStates.tsx [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 import { userLogic } from 'scenes/userLogic'
 import { IllustrationDanger } from 'lib/components/icons'
 
-export function LineGraphEmptyState({ color }: { color: string }): JSX.Element {
-    const { featureFlags } = useValues(featureFlagLogic)
+export function LineGraphEmptyState({ color, isDashboard }: { color: string; isDashboard?: boolean }): JSX.Element {
     return (
         <>
-            {!featureFlags['1694-dashboards'] && (
+            {!isDashboard ? (
```
Comment (yakkomajuri) [inline frontend/src/scenes/insights/EmptyStates.tsx:12]: super nit: generally nicer to have the conditional not be a negation

File: posthog/test/test_demo.py [hunk 2]
```diff
@@ -12,12 +12,15 @@ def test_create_demo_data(self):
...
+        self.assertEqual(demo_team.is_demo, True)
+        self.assertEqual(Dashboard.objects.count(), 3)
         self.assertGreaterEqual(Event.objects.count(), 900)
         self.assertGreaterEqual(Person.objects.count(), 160)
         self.assertGreaterEqual(Action.objects.count(), 8)
         self.assertGreaterEqual(SessionRecordingEvent.objects.count(), 60)
 
-        action_event_counts = [action.events.count() for action in Action.objects.all()]
-        self.assertCountEqual(action_event_counts, [14, 140, 0, 0, 40, 100, 73, 87])
+        # TODO: We need a better way to test this, inconsistent results locally and on CI
```
Comment (yakkomajuri) [inline posthog/test/test_demo.py:22]: agreed

File: frontend/src/scenes/insights/EmptyStates.tsx [hunk 3]
```diff
@@ -27,7 +24,7 @@ export function LineGraphEmptyState({ color }: { color: string }): JSX.Element {
                         Seems like there's no data to show this graph yet{' '}
                         <a
                             target="_blank"
-                            href="https://posthog.com/docs/features/trends"
+                            href="https://posthog.com/docs/features/trends?utm_campaign=dashboard-empty-state&utm_medium=in-product"
```
Comment (yakkomajuri) [inline frontend/src/scenes/insights/EmptyStates.tsx:27]: :+1:

**Review by yakkomajuri — APPROVED** [review]
Gave this a quick look, looks good to me. Added a tiny nit.

**Comment by paolodamico** [comment]
Thanks for the review @yakkomajuri !

**Pull request merged**